Makefile

```diff
@@ -1,44 +1,62 @@
-clean:
+ifeq ($(UNAME),Darwin)
+     SHELL := /opt/local/bin/bash
+     OS_X  := true
+ else ifneq (,$(wildcard /etc/redhat-release))
+     OS_RHEL := true
+ else
+     OS_DEB  := true
+     SHELL := /bin/bash
+ endif 
```

Comment:
This wasn't required under Darwin and other projects with make help I've worked on didn't bother with this or just set `SHELL := /bin/bash`